application-services/custom/fledge-export/main.go

```diff
@@ -0,0 +1,59 @@
+package main
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/edgexfoundry/app-functions-sdk-go/appsdk"
+	"github.com/edgexfoundry/app-functions-sdk-go/pkg/transforms"
+
+	fledgeTransforms "fledge-export/pkg/transforms"
+)
+
+func main() {
+
+	// 1) First thing to do is to create an instance of the EdgeX SDK, giving it a service key
+	edgexSdk := &appsdk.AppFunctionsSDK{
+		ServiceKey: "FledgeExport", // Key used by Registry (Aka Consul)
+	}
+
+	// 2) Next, we need to initialize the SDK
+	if err := edgexSdk.Initialize(); err != nil {
+		message := fmt.Sprintf("SDK initialization failed: %v\n", err)
+		if edgexSdk.LoggingClient != nil {
+			edgexSdk.LoggingClient.Error(message)
+		} else {
+			fmt.Println(message)
+		}
+		os.Exit(-1)
```

Comment:
This is the old code. `LoggingClient` will now never be nil. See other examples:
https://github.com/edgexfoundry/edgex-examples/blob/master/application-services/custom/simple-filter-xml/main.go#L40